Source/WebKit/WebProcess/WebProcess.messages.in

```diff
@@ -242,5 +242,6 @@ messages -> WebProcess : AuxiliaryProcess WantsAsyncDispatchMessage {
 
 #if PLATFORM(COCOA)
     RegisterAdditionalFonts(struct WebKit::AdditionalFonts fonts)
+    RegisterFontMap(HashMap<String, URL> fonts, Vector<WebKit::SandboxExtensionHandle> sandboxExtensions)
```

Comment:
`WebKit::` likely unnecessary

Reply:
It looks like we need this in `messages.in` files.

Reply:
Thanks for reviewing!